drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1715,6 +1715,12 @@ function DragAndDropBlock(runtime, element, configuration) {
                     state.grade = data.grade;
                     if (!data.correct) {
                         delete state.items[item_id];
+                        var failure_audio = new Audio('https://pakistanx-static-assets.s3.amazonaws.com/wrong.mp3');
```

Comment:
I would suggest creating audio instances on page load & play when required instead of creating new ones on each event.

Reply:
Done!